src/server/api/routers/login.ts

```diff
@@ -9,44 +9,48 @@ import { resetPasswordTokens, users } from "~/server/db/schema";
 import { eq } from "drizzle-orm";
 import { TRPCError } from "@trpc/server";
 import { resetTemplate } from "./password-reset-template";
+import { authOptions } from "~/server/auth";
+import { Adapter } from "next-auth/adapters";
 
 const TOKEN_EXPIRY = 1000 * 60 * 10; // 10 minutes
 
 export const loginRouter = createTRPCRouter({
```

Comment:
Would you be able to rename this router to authRouter, change this file to be called auth.ts, and make the same change in `api/root.ts` as well (login -> auth)?